Dockerfile

```diff
@@ -22,7 +22,8 @@ WORKDIR /etc/opt/goat/
 ENV PIP_NO_CACHE_DIR=1
 COPY Pipfile Pipfile.lock ./
 # hadolint ignore=DL3016,DL3018
-RUN pipenv install --deploy --ignore-pipfile \
+RUN pip install pipenv==2022.5.2 \
```

Comment:
Because these kind of dependencies can easily become unmaintained, I prefer either to not hard code them, or make sure that `invoke update` will update them.